src/main/kotlin/com/chattriggers/ctjs/utils/CategorySorting.kt

```diff
@@ -0,0 +1,13 @@
+package com.chattriggers.ctjs.utils
+
+import gg.essential.vigilance.data.Category
+import gg.essential.vigilance.data.SortingBehavior
+
+object CategorySorting: SortingBehavior() {
+    override fun getCategoryComparator(): Comparator<in Category> {
```

Comment:
Lets make this throw if a category isn't in the list, that way if another category gets added in the future the implementer won't forget to add it here.